core/src/test/java/org/apache/calcite/test/JdbcFrontLinqBackTest.java

```diff
@@ -343,12 +356,13 @@ private CalciteAssert.AssertThat mutable(
         .with(makePostProcessor(employees));
   }
 
-  static AbstractModifiableTable mutable(String tableName,
```

Comment:
It doesn't seem right to pass a type factory into a table constructor. I know these are only mock tables for testing. But tables have a lifetime longer than a single statement, therefore longer than a type factory. That's why `getRowType` takes a type factory as an argument.

Reply:
Thanks for the review @julianhyde! I apologize for the delay. I got rid of the strange `postProcessor` business in favor of a `withMeasures` flag. May not be ideal but is easier to follow than the passing a type factory for the table to use.